src/Core/src/Platform/Windows/WindowManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Maui.Graphics;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace Microsoft.Maui
+{
+	public class WindowManager
```

Comment:
Do we want this public already?

Reply:
Yea, might as well make it public now so users can play with it and give feedback